docs/source/building.rst

```diff
@@ -0,0 +1,112 @@
+.. _building:
+
+Building PYJNIus
+================
+
+Building PyJNIus depends on `Cython <http://cython.org/>`_ and the `Java
+<http://www.oracle.com/javase>`_ Development Kit (includes the Java Runtime
+Environment).
+
+
+Building on GNU/Linux distributions
+-----------------------------------
+
+You need the GNU Compiler Collection (GCC), the JDK installed (openjdk
+will do), and Cython. Then, just type::
```

Comment:
My recollection is that doing pip creates a venv, so Cython need not be installed these days?

Reply:
Yes, you're right. `pip install .` or `pip install -e .` will install build deps (like `Cython`) in a separate build-only environment, so it's not needed anymore.